modules/influxdb/docker-entrypoint-initdb.d/test_v1_data.sh

```diff
@@ -0,0 +1 @@
+influx -import -path=/docker-entrypoint-initdb.d/NOAA_data.txt.gz -compressed -precision=s -database=NOAA_water_database
```

Comment:
Is this test script and its data needed by the module or only used in tests? If the latter, please add it to a `testdata` directory and copy it into the container. Otherwise all users of the module will receive the files in here